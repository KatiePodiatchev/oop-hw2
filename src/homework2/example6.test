# Should add edges to the graph

# Create a graph
CreateGraph graph1

# Create and add nodes
CreateNode A 2
CreateNode B 1
CreateNode C 3
CreateNode D 1
CreateNode E 4
AddNode graph1 A
AddNode graph1 B
AddNode graph1 C
AddNode graph1 D
AddNode graph1 E

# Add edges
AddEdge graph1 A B
AddEdge graph1 A C
AddEdge graph1 B C
AddEdge graph1 B D
AddEdge graph1 C D
AddEdge graph1 C E
AddEdge graph1 D E

# Find a path
FindPath graph1 A -> E
FindPath graph1 E -> A B C D
FindPath graph1 A B D E -> C
FindPath graph1 A B C D E -> A B C D E