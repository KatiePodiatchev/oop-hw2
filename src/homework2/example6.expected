# Should add edges to the graph

# Create a graph
created graph graph1

# Create and add nodes
created node A with cost 2
created node B with cost 1
created node C with cost 3
created node D with cost 1
created node E with cost 4
added node A to graph1
added node B to graph1
added node C to graph1
added node D to graph1
added node E to graph1

# Add edges
added edge from A to B in graph1
added edge from A to C in graph1
added edge from B to C in graph1
added edge from B to D in graph1
added edge from C to D in graph1
added edge from C to E in graph1
added edge from D to E in graph1

# Find a path
shortest path in graph1: A B D E
There is no path in graph1
shortest path in graph1: B C
shortest path in graph1: B
